src/backend/ceph.rs

```diff
@@ -578,13 +622,7 @@ impl CephBackend {
         }
         let osd_id = osd_id.unwrap();
         debug!("Toggle noscrub, nodeep-scrub flags");
-        osd_set(&self.cluster_handle, &OsdOption::NoScrub, false, simulate)?;
-        osd_set(
-            &self.cluster_handle,
-            &OsdOption::NoDeepScrub,
-            false,
-            simulate,
-        )?;
+        self.set_noscrub(simulate)?;
```

Comment:
If the noscrub flag is already on, would this turn it off?

Reply:
Nope.  Just runs set.  If you're setting a flag that's already set it does doesn't do anything lol